TodoList/Tests/TodoList.Api.Tests/Controllers/ItemsControllerTest.cs

```diff
@@ -70,37 +81,40 @@ public async Task GetAsync_WithInvalidId_ReturnsNoItem()
             var id = new Guid("5E73F108-97F6-4FFF-A15C-1A7AEDE686BA");
 
             var contentResult = await _controller
-                .ExecuteAction(controller => controller.GetAsync(id));
+                .ExecuteAction(controller => controller.GetItemAsync(id));
 
             Assert.That(contentResult.StatusCode, Is.EqualTo(HttpStatusCode.NotFound));
         }
 
         [Test]
         public async Task PostAsync_AddOneItem_ReturnsAddedItem()
         {
-            var inputItem = new Item {Text = "CatDog"};
-            _controller.Request.RequestUri = new Uri("http://location/items");
+            _repository.AddItemAsync(s_catDog).Returns(s_items[0]);
+            var uriString = $"http://location/objects/{s_items[0].Id}/CreatedPath";
```

Comment:
```suggestion
            var uriString = new Uri($"http://location/objects/{s_items[0].Id}/CreatedPath");
```
Then you can easilly test id uri is the same as result (without using `ToString()`).

Reply:
2dde231 + b4789a7